### ISSUE TITLE
Where is Minimac4-omp ?

### ISSUE THREAD

**Issue by rozaimirazali** [statgen/Minimac4#3]
Hi. I was wondering if the current version of Minimac4 contains the omp capability?

If not, how can it do multi-threading?

Thanks

**Comment by Santy-8128** [comment]
Yes, by default it is multi-threaded and uses 5 CPUs. If you want to run a
single thread version use --cpu 1

*Regards,*
*Sayantan Das,*



*Post-Doctoral Fellow Department of BiostatisticsUniversity of Michigan,
Ann Arbor.MI 48109-2029*
